packages/react-select/src/Select.js

```diff
@@ -808,17 +942,35 @@ export default class Select extends Component<Props, State> {
   getElementId = (element: 'group' | 'input' | 'listbox' | 'option') => {
     return `${this.instancePrefix}-${element}`;
   };
-  getActiveDescendentId = () => {
-    const { menuIsOpen } = this.props;
-    const { menuOptions, focusedOption } = this.state;
 
-    if (!focusedOption || !menuIsOpen) return undefined;
+  getComponents = () => {
+    return defaultComponents(this.props);
+  };
 
-    const index = menuOptions.focusable.indexOf(focusedOption);
-    const option = menuOptions.render[index];
+  buildCategorizedOptionsFromPropsAndSelectValue = memoizeOne(
+    buildCategorizedOptions,
+    (newArgs: any, lastArgs: any) => {
+      const [newProps, newSelectValue] = (newArgs: [Props, OptionsType]);
+      const [lastProps, lastSelectValue] = (lastArgs: [Props, OptionsType]);
```

Comment:
This method is memoized per instance of this component, however since `getDerivedStateFromProps` is `static`, there is no way to memoize the call that `getDerivedStateFromProps` makes to `buildCatgeroizedOptions` and have it behave the same with multiple instances as it does with one instance.